### ISSUE TITLE
📝 Invalid document: Expected end tag of kind Fill but found Indent.

### ISSUE THREAD

**Issue by XhstormR** [biomejs/biome#8437]
### Detail

Source code: https://github.com/XhstormR/p2p/blob/main/src/app/message/message.component.html

```shell
message.component.html format ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Invalid document: Expected end tag of kind Fill but found Indent.

     This is an internal Biome error. Please report if necessary.
```

### Environment information

```bash
CLI:
  Version:                      2.3.8
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           macos

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  BIOME_THREADS:                unset
  NO_COLOR:                     unset
  TERM:                         tmux-256color
  JS_RUNTIME_VERSION:           v25.2.1
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         bun/1.3.4

Biome Configuration:
  Status:                       Loaded successfully
  Path:                         /Users/user/Documents/IdeaProjects/p2p/biome.json
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               true
  VCS enabled:                  true

Formatter:
  Format with errors:           unset
  Indent style:                 Space
  Indent width:                 4
  Line ending:                  LF
  Line width:                   110
  Attribute position:           unset
  Bracket spacing:              unset
  Includes:                     unset

JavaScript Formatter:
  Enabled:                      unset
  JSX quote style:              unset
  Quote properties:             unset
  Trailing commas:              unset
  Semicolons:                   unset
  Arrow parentheses:            As needed
  Bracket spacing:              unset
  Bracket same line:            unset
  Quote style:                  unset
  Indent style:                 unset
  Indent width:                 unset
  Line ending:                  unset
  Line width:                   unset
  Attribute position:           unset

JSON Formatter:
  Enabled:                      unset
  Indent style:                 unset
  Indent width:                 unset
  Line ending:                  unset
  Line width:                   unset
  Trailing Commas:              unset
  Expand lists:                 unset

CSS Formatter:
  Enabled:                      unset
  Indent style:                 unset
  Indent width:                 unset
  Line ending:                  unset
  Line width:                   unset
  Quote style:                  unset

GraphQL Formatter:
  Enabled:                      unset
  Indent style:                 unset
  Indent width:                 unset
  Line ending:                  unset
  Line width:                   unset
  Bracket spacing:              unset
  Quote style:                  unset

Workspace:
  Open Documents:               0
```

### Configuration

```JSON
{
    "$schema": "https://biomejs.dev/schemas/2.3.8/schema.json",
    "vcs": {
        "enabled": true,
        "clientKind": "git",
        "useIgnoreFile": true
    },
    "files": {
        "ignoreUnknown": false
    },
    "formatter": {
        "enabled": true,
        "lineWidth": 110,
        "lineEnding": "lf",
        "indentWidth": 4,
        "indentStyle": "space"
    },
    "javascript": {
        "formatter": {
            "arrowParentheses": "asNeeded"
        }
    },
    "html": {
        "formatter": {
            "enabled": true
        },
        "parser": {
            "interpolation": true
        }
    },
    "linter": {
        "enabled": true,
        "rules": {
            "recommended": true
        }
    },
    "assist": {
        "enabled": true,
        "actions": {
            "source": {
                "organizeImports": "on"
            }
        }
    }
}
```

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by dyc3** [comment]
This has been fixed on `next`.
